notebooks/CubevizExample.ipynb

```diff
@@ -61,7 +64,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "fn = download_file('https://stsci.box.com/shared/static/28a88k1qfipo4yxc4p4d40v4axtlal8y.fits', cache=True)\n",
+    "uri = \"mast:JWST/product/jw02732-c1001_t004_miri_ch1-longmediumshort-_s3d.fits\"\n",
+    "result = Observations.download_file(uri, base_url=\"https://mast.stsci.edu/api/v0.1/Download/file\")\n",
```

Comment:
Doesn't think create a cache dir under the working directory? We need to add that to `.gitignore`.

Reply:
It doesn't, it just downloads the files to the working directory.